vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -663,15 +676,355 @@ declare class VraNetworkInterfaceSpecification {
+	id: string;
+	isGlobalManager: boolean;
+	updatedAt: string;
+	username: string;
+	constructor();
+
+	/**
+	 * @param key
+	 * @param linksItem
+	 */
+	putLinksItem(key: string, linksItem: VraHref): VraCloudAccountNsxT;
+
+	/**
+	 * @param key
+	 * @param customPropertiesItem
+	 */
+	putCustomPropertiesItem(
+		key: string,
+		customPropertiesItem: string
+	): VraCloudAccountNsxT;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraCloudAccountNsxT;
+}
+
+declare class VraCloudAccountNsxTResult {
+	readonly numberOfElements: number;
+	readonly content: Object[];
+	readonly totalElements: number;
+	constructor();
+}
+
+/**
+ * State object representing an Nsx-V cloud account.<br><br>A cloud account identifies a cloud account type and an account-specific deployment region or data center where the associated cloud account resources are hosted.<br>**HATEOAS** links:<br>**self** - CloudAccountNsxV - Self link to this cloud account
+ */
+declare class VraCloudAccountNsxV {
+	owner: string;
+	hostName: string;
+	description: string;
+	orgId: string;
+	tags: Object[];
+	createdAt: string;
+	dcid: string;
+	readonly internalIdString: string;
+	name: string;
+	host: VraHost;
+	id: string;
+	updatedAt: string;
+	username: string;
+	constructor();
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraCloudAccountNsxV;
+
+	/**
+	 * @param key
+	 * @param customPropertiesItem
+	 */
+	putCustomPropertiesItem(
+		key: string,
+		customPropertiesItem: string
+	): VraCloudAccountNsxV;
+
+	/**
+	 * @param key
+	 * @param linksItem
+	 */
+	putLinksItem(key: string, linksItem: VraHref): VraCloudAccountNsxV;
+}
+
+/**
+ * State object representing a query result of Nsx-V cloud accounts.
+ */
+declare class VraCloudAccountNsxVResult {
+	readonly numberOfElements: number;
+	readonly content: Object[];
+	readonly totalElements: number;
+	constructor();
+}
+
+/**
+ * State object representing cloud account region.<br><br>**externalRegions** - array[RegionSpecification] - Set of regions that can be enabled for this cloud account.<br>**externalRegionIds** - array[String] - Set of ids of regions that can be enabled for this cloud account.<br>
+ */
+declare class VraCloudAccountRegions {
+	externalRegions: Object[];
+	constructor();
+
+	/**
+	 * @param externalRegionsItem
+	 */
+	addExternalRegionsItem(
+		externalRegionsItem: VraRegionSpecification
+	): VraCloudAccountRegions;
+}
+
+declare class VraRegion {
+	owner: string;
+	createdAt: string;
+	externalRegionId: string;
+	readonly internalIdString: string;
+	cloudAccountId: string;
+	name: string;
+	host: VraHost;
+	id: string;
+	orgId: string;
+	updatedAt: string;
+	constructor();
+
+	/**
+	 * @param key
+	 * @param linksItem
+	 */
+	putLinksItem(key: string, linksItem: VraHref): VraRegion;
+}
+
+/**
+ * State object representing a query result of cloud accounts.
+ */
+declare class VraCloudAccountResult {
+	readonly numberOfElements: number;
+	readonly content: VraCloudAccount[];
```

Comment:
Just a note, in the API explorer the return type is Array/Object. But if you tested it and VraCloudAccount[] works, I would say let's keep it like this since specific type is better then a generalised "Object" type
![image](https://github.com/user-attachments/assets/df98e9f0-7906-4c13-a67a-82fcefd101ac)